src/ameba.ts

```diff
@@ -58,7 +66,7 @@ export class Ameba {
                 outputChannel.appendLine(`$ ${args.join(' ')}`)
                 const proc = spawn(args[0], args.slice(1), { cwd: dir });
 
-                if (virtual) {
+                if (virtual && !isWorkspace) {
```

Comment:
This additional check doesn't have much sense, given `if (isWorkspace) virtual = false` on line 35. 

```suggestion
                if (virtual) {
```

Reply:
The check isn't necessary logically, but it is necessary so that the ts compiler knows that document within this block is only TextDocument